### PR TITLE
#72 - Change meetup page design

### LINKED ISSUE #72: Change Meetup page design
Meetup page need to be redesign in section with tags and links to yt and slideshare.

### PR DESCRIPTION
**Pull request by paulinaszkudlarek**
This should close #72 

image
image

Mobile:
image
image
image

### CODE REVIEW

**Review by EwelinaSkrzypacz — COMMENTED** [review]
1. Why after hover on last presentation I see white space?

image

2. When there is no links to presentation, speaker picture on center looks (for me) lonely ;p

image

3. On meetups list I will remove comma from last tag or maybe tags should look like on the meetup details page?

image

**Comment by paulinaszkudlarek** [comment]
@EwelinaLasowy 
1. Resolved
2. wip :)
3. Resolved, tags looks now like ona  meetup page

**Review by EwelinaSkrzypacz — DISMISSED** [review]

**Review by EwelinaSkrzypacz — COMMENTED** [review]
Please change favico format to `.ico`.

**Review by EwelinaSkrzypacz — DISMISSED** [review]

File: index.html [hunk 1]
```diff
@@ -4,6 +4,7 @@
     <meta charset="utf-8">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <link href='https://api.mapbox.com/mapbox-gl-js/v2.12.0/mapbox-gl.css' rel='stylesheet' />
+    <link rel="icon" href="/public/images/favicon.ico">
```
Comment (MichalMyskow) [inline index.html]:
```suggestion
    <link rel="icon" href="/images/favicon.ico">
```
image

File: frontend/components/Meetup/PresentationsList.vue [hunk 2]
```diff
@@ -9,88 +9,96 @@ defineProps({
...
+  },
 })
 
 </script>
 
 <template>
-  <div class="mt-12 overflow-hidden bg-white text-zinc-700">
-    <div class="border-b border-zinc-200 px-4 py-5 sm:px-6">
-      <h3 class="pl-4 text-xl font-medium leading-8">
+  <div v-if="!loading" class="mt-12 bg-white text-zinc-700 shadow-lg">
```
Comment (MichalMyskow) [inline frontend/components/Meetup/PresentationsList.vue]: Do we need that **loading** here?
We are already hiding this component in `Meetup` component.

```
<div v-if="!loading">
  <meetup-header :loading="loading" :meetup="meetup"/>
  <presentations-list :loading="loading" :presentations="meetup.presentations"/>
</div>
```

File: frontend/components/Meetup/MeetupHeader.vue [hunk 3]
```diff
@@ -8,31 +8,38 @@ defineProps({
...
+  loading: {
+    type: Boolean, 
+    default: true,
+  },
 })
 </script>
 
 <template>
-  <div class="relative overflow-hidden bg-white">
+  <div v-if="!loading" class="relative overflow-hidden bg-white shadow-lg">
```
Comment (MichalMyskow) [inline frontend/components/Meetup/MeetupHeader.vue]: Same as with presentations

File: frontend/components/Meetup/PresentationsList.vue (hunk 2 above)
Comment (paulinaszkudlarek) [reply frontend/components/Meetup/PresentationsList.vue]: You're right, I just commited a change for that

File: frontend/components/Meetup/MeetupHeader.vue (hunk 3 above)
Comment (paulinaszkudlarek) [reply frontend/components/Meetup/MeetupHeader.vue]: Just commited changes as well

File: frontend/pages/Meetup.vue [hunk 4]
```diff
@@ -37,7 +37,7 @@ onMounted(() => {
 <template>
   <LoadingSpinner v-if="loading"/>
   <div v-if="!loading">
-    <meetup-header :meetup="meetup"/>
-    <presentations-list :presentations="meetup.presentations"/>
+    <meetup-header :loading="loading" :meetup="meetup"/>
+    <presentations-list :loading="loading" :presentations="meetup.presentations"/>
```
Comment (MichalMyskow) [inline frontend/pages/Meetup.vue]:
```suggestion
    <meetup-header :meetup="meetup"/>
    <presentations-list :presentations="meetup.presentations"/>
```

**Review by MichalMyskow — APPROVED** [review]

**Review by EwelinaSkrzypacz — APPROVED** [review]

**Pull request merged**